nomad-gcp/main.tf

```diff
@@ -123,6 +123,7 @@ data "google_compute_image" "ubuntu_2004" {
 resource "google_compute_firewall" "default" {
   name    = "allow-retry-with-ssh-circleci-server-${var.name}"
   network = var.network
+  project = var.host_project != "" ? var.host_project : var.project
```

Comment:
Perhaps something like this (completely untested and my regex is probably wrong, but you get the idea) would allow us to remove the var.host_project variable entirely

```suggestion
  project = length(regexall("projects/(.*)/regions/", var.subnet)) > 0 ? regex("projects/(.*)/regions/") : var.project
```

Reply:
I couldn't quite get it working like how you suggested so i reverted to a version i know works. Idk if anyone wants to take a whack at it